docs/pages/versions/unversioned/config/metro.mdx

```diff
@@ -589,6 +589,14 @@ The `debugId` is a deterministic hash of the bundle's contents without the exter
 
 `@expo/metro-config` injects `debugId` during `npx expo export` and `npx expo export:embed`. Any additional optimization steps in `npx expo export:embed` like Hermes bytecode generation will need to have the `debugId` injected manually.
 
+## Metro require runtime
+
+In SDK 52, you can optionally enable a custom Metro `require` implementation with the environment variable `EXPO_USE_METRO_REQUIRE=1`. This runtime has the following features:
```

Comment:
```suggestion
You can optionally enable a custom Metro `require` implementation with the environment variable `EXPO_USE_METRO_REQUIRE=1`. This runtime has the following features:
```

We remove SDK 52 part since this info is going into unversioned/Next docs as of now and will be part of SDK 52 release API docs. Not going to be visible for older SDKs.